app/views/layouts/footer.html.erb

```diff
@@ -0,0 +1,65 @@
+<section class="return-to-top">
+  <div class="grid-container grid-container-desktop usa-footer__return-to-top">
+    <div class="grid-row">
+      <div class="grid-col-auto">
+        <a href="#"><span>Return to top</span> <i class="fas fa-arrow-circle-up"></i></a>
```

Comment:
This icon doesn't show up for me -- I think because we are not actually pulling the FontAwesome library or using it anywhere else.